core-feature-pack/common/src/main/resources/content/bin/common.bat

```diff
@@ -40,46 +40,41 @@ goto :eof
 :setDefaultModularJvmOptions
   call :setModularJdk
   if "!MODULAR_JDK!" == "true" (
-    echo "%~1" | findstr /I "\-\-add\-modules" > nul
```

Comment:
This seems to no longer serve its purpose.
And it's probably not even working as supposed, currently (for some time) it's checking only against the first argument from JAVA_OPTS, and so it's always passing.

Reply:
Thanks @yersan 

> With your change, we will no longer able to check whether --add-modules is in JAVA_OPTS.
I have been too hasty with this one.

It seems that adding only quotation marks is not sufficient, as JAVA_OPTS is even then broken into multiple arguments, and then the check is not always relevant.
`echo %* | findstr /I "\-\-add\-modules" > nul`
should fix it.